bindinate/Makefile.am.template

```diff
@@ -9,13 +9,12 @@ GLUEDIR = $(srcdir)/glue
 SRC_FILES_XML = $(srcdir)/src_files.xml
 CSPROJ_XSLT = $(srcdir)/csproj.xslt
 GEN_SOURCES = $(srcdir)/generated.sources
+CUSTOM_SOURCES = $(srcdir)/custom.sources
 
 gapidir = $(GAPIXMLDIR)
 gapi_DATA = $(API)
 
-# Add any extra source files you need here
-sources =
-
+sources = $(shell grep -Ev '\#.*$$' $(CUSTOM_SOURCES))
```

Comment:
I don't think so. A included makefile would need to look like
CUSTOM_SOURCES = \
     File1.cs \
     File2.cs